models/br_sp_saopaulo_geosampa_iptu/code/br_sp_saoupaulo_geosampa_iptu.py

```diff
@@ -0,0 +1,229 @@
+                header=not file_filter_save_path.exists(),
+            )
+    else:
+        raise BaseException("Data need to be a pandas DataFrame")
+
+for anos in anos_tratamento:
+    print(f'Abrindo o arquivo de {anos}')
+    df = pd.read_csv(f'D:\download\iptu\EG_{anos}.csv', sep=';', encoding='utf-8')
+
+    rename = {'NUMERO DO CONTRIBUINTE':'numero_contribuinte',
+    'ANO DO EXERCICIO':'ano',
+    'NUMERO DA NL':'numero_notificacao',
+    'DATA DO CADASTRAMENTO':'data_cadastramento',
+    'NUMERO DO CONDOMINIO':'numero_condominio',
+    'CODLOG DO IMOVEL':'codigo_logradouro',
+    'NOME DE LOGRADOURO DO IMOVEL':'logradouro',
+    'NUMERO DO IMOVEL':'numero_imovel',
+    'COMPLEMENTO DO IMOVEL':'complemento',
+    'BAIRRO DO IMOVEL':'bairro',
+    'REFERENCIA DO IMOVEL':'referencia_imovel',
+    'CEP DO IMOVEL':'cep',
+    'TIPO DE USO DO IMOVEL':'finalidade_imovel',
+    'TIPO DE PADRAO DA CONSTRUCAO':'tipo_construcao',
+    'TIPO DE TERRENO':'tipo_terreno',
+    'QUANTIDADE DE ESQUINAS FRENTES':'quantidade_esquina_imovel',
+    'FRACAO IDEAL':'fracao_ideal',
+    'AREA DO TERRENO':'area_terreno',
+    'AREA CONSTRUIDA':'area_construida',
+    'AREA OCUPADA':'area_ocupada',
+    'VALOR DO M2 DO TERRENO':'valor_terreno',
+    'VALOR DO M2 DE CONSTRUCAO':'valor_construcao',
+    'ANO DA CONSTRUCAO CORRIGIDO':'ano_construcao_corrigida',
+    'QUANTIDADE DE PAVIMENTOS':'quantidade_pavimento',
+    'TESTADA PARA CALCULO':'testada_imovel',
+    'ANO DE INICIO DA VIDA DO CONTRIBUINTE':'ano_inicio_vida_contribuinte',
+    'MES DE INICIO DA VIDA DO CONTRIBUINTE':'mes_inicio_vida_contribuinte',
+    'FASE DO CONTRIBUINTE':'fase_contribuinte',
+    'FATOR DE OBSOLESCENCIA':'fator_obsolescencia'}
+
+    ordem = ['ano',
+    'data_cadastramento',
+    'numero_contribuinte',
+    'numero_notificacao',
+    'numero_condominio',
+    'codigo_logradouro',
+    'logradouro',
+    'numero_imovel',
+    'cep',
+    'complemento',
+    'bairro',
+    'referencia_imovel',
+    'finalidade_imovel',
+    'tipo_construcao',
+    'tipo_terreno',
+    'fracao_ideal',
+    'area_terreno',
+    'area_construida',
+    'area_ocupada',
+    'testada_imovel',
+    'quantidade_esquina_imovel',
+    'valor_terreno',
+    'valor_construcao',
+    'ano_construcao_corrigida',
+    'quantidade_pavimento',
+    'ano_inicio_vida_contribuinte',
+    'mes_inicio_vida_contribuinte',
+    'fase_contribuinte',
+    'fator_obsolescencia']
+
+
+    print('renomenado...')
+    df.rename(columns=rename, inplace=True)
+    print('tratando...')
+    lista = ['fator_obsolescencia',
+            'testada_imovel',
+            'valor_construcao',
+            'valor_terreno',
+            'fracao_ideal',
+            ]
+    for x in lista:
+        df[x] = df[x].apply(lambda x: str(x).replace(',', '.'))
+
+    cep_codigo_logradouro = ['cep',
+                            'codigo_logradouro',
+                            'numero_contribuinte']
+    for traço in cep_codigo_logradouro:
```

Comment:
nesse caso é uma boa prática que fique claro o que é o 'traço' e 'cep_codigo_logradouro', algo como 

`for coluna in colunas_remove_traco:`